app/controllers/concerns/content_data_helper.rb

```diff
@@ -0,0 +1,16 @@
+module ContentDataHelper
+  # Helper method to create hash for any part of a component or partial which needs a content and data hash to send through a translation key with multiple data items.
+  # The data hash contains key/value pairs where the key is also the key inside curly braces in the string to be interpolated. The values are the data from the database which will be inserted into the string.
+  # @param [String] content is the translation key to be added as the content value
+  # @param [Symbol] key is the key
```

Comment:
There are only two params to document: content and **data.  You could do something like:
`@param [String] content ...`
`@params [Array<Hash>] data ...`